changelog/changelog.py

```diff
@@ -158,7 +158,7 @@ def default_changelog_generator(items):
     lines = []
     for item in items:
         title = item['title']
-        breaks_compat = 'compatibility' in item['labels']
+        breaks_compat = any(l['name'] == 'compatibility' for l in item['labels'])
```

Comment:
Ahhhh! `item['labels']` is a _list of dicts_, so `item['labels']` will never contain the string `'compatibility'`.
 
Good catch! No wonder I've never seen the "breaks compatibility" warning in any changelogs! :sweat:

Reply:
To be fair, the API accepts a list of dict-or-string objects, but it seems to produce a strict list of dict objects. Maybe an earlier version produced a list of strings?